src/src.pro

```diff
@@ -142,8 +142,9 @@ win32 {
 }
 mac {
     RC_FILE = macapp.icns
-    INCLUDEPATH += -I/usr/local/include \
-                   -I/usr/local/opt/sqlite/include
+    QT+= macextras
+    INCLUDEPATH += /usr/local/include \
```

Comment:
Yeah, that's what I remember seeing. :wink:

I've just installed Yosemite in a VM, so am going through getting Homebrew functional so I can test this.